packages/experimental/TabList/src/Tab/Tab.tsx

```diff
@@ -50,6 +52,7 @@ export const Tab = compose<TabType>({
         return null;
       }
 
+      // Only support text as content for now.
       let text = '';
       React.Children.forEach(children, (child) => {
         if (typeof child === 'string') {
```

Comment:
What does this do? It seems to set text to whatever the last child is?

Reply:
This gets the text for the accessibility label. For the text content in the Tab button, I want text to render similar to how the FURN button does it, but comparing code, I'm realizing my mistake. Let me fix this.